dynawo/sources/Simulation/DYNSimulation.cpp

```diff
@@ -498,32 +484,9 @@ Simulation::loadDynamicData() {
   dyd_->setParametersReference(referenceParameters_);
 
   if (iidmFile_ != "") {
-    try {
```

Comment:
I am very please with this it is way cleaner and was bothering me for quite some time